src/mutator.rs

```diff
@@ -247,29 +247,9 @@ impl Scope {
         });
     }
 
-    /// Deferred deallocation of heap-allocated object `ptr`.
-    ///
-    /// This function inserts the object into a mutator-local [`Bag`]. When the bag becomes full,
-    /// the bag is flushed into the globally shared queue of bags.
-    ///
-    /// If the object is unusually large, it is wise to follow up with a call to [`flush`] so that
-    /// it doesn't get stuck waiting in the local bag for a long time.
-    ///
-    /// [`Bag`]: struct.Bag.html
-    /// [`flush`]: fn.flush.html
-    pub unsafe fn defer_free<T>(&self, ptr: Ptr<T>) {
-        self.defer_garbage(Garbage::new_free(ptr.as_raw() as *mut T, 1))
-    }
-
-    /// Deferred destruction and deallocation of heap-allocated object `ptr`.
-    // FIXME(jeehoonkang): `T: 'static` may be too restrictive.
-    pub unsafe fn defer_drop<T: Send + 'static>(&self, ptr: Ptr<T>) {
-        self.defer_garbage(Garbage::new_drop(ptr.as_raw() as *mut T, 1))
-    }
-
     /// Deferred execution of an arbitrary function `f`.
```

Comment:
It would be good to readd the information from the comment on `defer_free` here. For example:
> This function adds a closure to be executed later into a mutator-local [`Bag`]. When the bag becomes full,
> it is flushed into the globally shared queue of bags.

> If the closure environment (captured variables, code) is unusually large, it is wise to follow up with a call
> to [`flush`] so that it doesn't get stuck waiting in the local bag for a long time.

Reply:
Readded.